libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -475,6 +475,15 @@ class MapboxNavigation(
         MapboxNativeNavigatorImpl.addHistoryEvent(eventType, eventJsonProperties)
     }
 
+    /**
+     * Update the route refresh interval.
```

Comment:
Specify in the docs that this interval is in milliseconds, and what is the default value if developers would like to go back.